src/Aquifer.Tiptap/Aquifer.Tiptap.esproj

```diff
@@ -0,0 +1,9 @@
+﻿<Project Sdk="Microsoft.VisualStudio.JavaScript.Sdk/1.0.1184077">
+  <PropertyGroup>
+    <BuildCommand>npm install &amp;&amp; npx esbuild node_modules/aquifer-tiptap/src/TiptapUtilities.ts --bundle --outfile=dist/TiptapUtilities.js --platform=node --target=node12</BuildCommand>
```

Comment:
The build command is copied from [aquifer-tiptap](https://github.com/BiblioNexusStudio/aquifer-tiptap/blob/e8971fe2a26b4f130b8f9d2cdfdafe8b17fd532c/package.json#L7).  I did have some issues when downgrading the "node" package in which case adding `node update aquifer-tiptap` in here fixed it but I didn't want to add that overhead to every build.  It's probably fine without it if we only increment the package.